Source/CesiumRuntime/Private/CesiumMetadataGenericValue.cpp

```diff
@@ -104,9 +104,16 @@ FCesiumMetadataArray UCesiumMetadataGenericValueBlueprintLibrary::GetArray(
     UPARAM(ref) const FCesiumMetadataGenericValue& Value) {
   return std::visit(
       [](auto value) -> FCesiumMetadataArray {
-        return CesiumMetadataConversions<
-            FCesiumMetadataArray,
-            decltype(value)>::convert(value, FCesiumMetadataArray());
+        auto createArrayView = [](const auto& array) -> FCesiumMetadataArray {
+          return FCesiumMetadataArray(array);
+        };
+
+        // TODO: Is this the best way?
```

Comment:
@kring I don't think the conversion system handled arrays, but it also didn't make much sense to define implicit conversions between arrays and non-array types. The old code crashed when trying to GetArray I believe. So I check if it is an array type and return an empty array otherwise. The array's component types can still be coerced using the convert code. Maybe this should be paired with an "IsArray" BP function.

That being said, the templating stuff always gets me a bit confused, so if you have a better idea let me know. Particularly if you think conversions of non-array types to array and vice-versa would make sense.

Reply:
Ah yeah looks like the old code would have just returned an empty `FMetadataArray` in all cases. I think it does make sense to convert any type to an array with one element of that type, but we can add that if/when we need it.